.github/workflows/e2e.upload-folder.schedule.yml

```diff
@@ -0,0 +1,155 @@
+name: schedule secure-upload-folder
+
+on:
+  # Daily run.
```

Comment:
Added this daily run because some of the new Action call other actions `@main`, so it we re to change something in these, we would not catch it during pre-submits.